### ISSUE TITLE
Port to WP8.1 Store

### ISSUE THREAD

**Issue by NPadrutt** [michelsalib/Gi7#93]
This would IMHO make a lot of sense since it allows us to make a bit a more vibrant UI because of a much better support for animations etc. Also it will allow us to create a Win8.1 version with relativ small effort if we wanna to do so. But anyway I expect here the most development in the comming versions of WP. 
I expect that we have the most work with the porting the gui. The logic should be largly compatible as is octokit. Since we wanna make the app translateable it would be good to make these changes together which will save a lot of effort.

**Comment by RandomlyKnighted** [comment]
I'm all for it. I'd also like to phase out the Windows Phone Toolkit UI components. Whenever we do port the app to WP 8.1 I think that would be a great time to phase out the toolkit and replace it with 8.1 components.

**Comment by NPadrutt** [comment]
sure thing, because the toolkit doesn't exist for WP8.1 store ;)
Also there would be no need anymore for the bindable appbar since the native app bar is bindable with WinPhone8.1 Store

**Comment by AlbertoMonteiro** [comment]
I guess we cant leave the 27.1% of users.
http://pt.slideshare.net/adduplex/wp-stats-20141223/6

**Comment by NPadrutt** [comment]
Well I expect to that this part will shrink more and more since all devices should get the update as far as I know.

**Comment by RandomlyKnighted** [comment]
Yeah my phone got the update about a week or two ago and Verizon is usually the last one to the Windows Phone party.

**Comment by NPadrutt** [comment]
Also a plan is to make the app stable and then pull the trigger. 8.0 can use a stable app and get the new as far as the get the update. Also we can further push fixes for the 8.0 version via a branch.

**Comment by AlbertoMonteiro** [comment]
I agree

**Comment by NPadrutt** [comment]
So, then first make it stable and then port? Or do we assume that this percentage will shrink fast enough?

**Comment by RandomlyKnighted** [comment]
With all the issues we've got open I'd feel better if we got a good stable app first then port to 8.1

**Comment by NPadrutt** [comment]
would it make sense to make a little release planning what we do wanna have in the next version?

**Comment by RandomlyKnighted** [comment]
Make sense to me, then that way we have a plan on what should be fixed first rather than just picking random bugs and fixing them.
